util/create-email.ts

```diff
@@ -15,12 +15,35 @@ console.log(
 );
 console.log(`SEND_EMAIL_FOR_MANDATARIS_EFFECTIEF SET TO: ${SEND_EMAILS}`);
 
-export async function sendMailTo(emailTo: string, mandataris) {
+export async function sendMissingBekrachtigingsmail(
+  emailTo: string,
+  mandatarissen,
+) {
+  if (mandatarissen.length == 0) {
+    return;
+  }
+  let mandatarisTable = '';
+  for (const mandataris of mandatarissen) {
+    mandatarisTable += `
+      <tr>
+        <td>${mandataris.name}</td>
+        <td>${mandataris.mandate}</td>
+      </tr>
+    `;
+  }
   const from = sparqlEscapeString(EMAIL_FROM_MANDATARIS_EFFECTIEF as string);
   const to = sparqlEscapeString(emailTo);
   const htmlMessage = `
     <p>Beste,</p>
-    <p>De mandataris ${mandataris.name} met mandaat ${mandataris.mandate} heeft al meer dan 10 dagen de publicatiestatus ‘Effectief’ zonder koppeling met een besluit.</p>
+    <p>Er zijn een aantal mandatarissen die al meer dan 10 dagen de publicatiestatus ‘Effectief’ hebben zonder koppeling met een besluit. Voor uw gemeente zijn dit er ${mandatarissen.length}.</p>
+    <p> Hieronder vindt u een overzicht van al deze mandatarissen: <p>
+    <table>
+      <tr>
+        <th>Naam</th>
+        <th>Mandaat</th>
+      </tr>
+      ${mandatarisTable}
+    </table>
```

Comment:
Does the table looks a little funky for you to? 
![image](https://github.com/user-attachments/assets/2cc15e1c-a37c-41a3-b739-81c1963c09a9)

Reply:
Aligned the headers to the left and added some more spacing
![image](https://github.com/user-attachments/assets/2e85be61-bf2d-4ade-a0d4-a39be193d09b)